tests/e2e/vmservice_utils.go

```diff
@@ -725,3 +724,54 @@ func wait4Vm2ReachPowerStateInSpec(
 	framework.Logf("VM %s reached the power state %v requested in the spec", vm.Name, vm.Spec.PowerState)
 	return vm, waitErr
 }
+
+// createVmServiceVmWithPvcsWithZone creates VM via VM service with given ns, sc, vmi, pvc(s) and bootstrap data for
+// cloud init on given zone
+func createVmServiceVmWithPvcsWithZone(ctx context.Context, c ctlrclient.Client, namespace string, vmClass string,
+	pvcs []*v1.PersistentVolumeClaim, vmi string, storageClassName string, secretName string,
+	zone string) *vmopv1.VirtualMachine {
+
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	vols := []vmopv1.VirtualMachineVolume{}
+	vmName := fmt.Sprintf("csi-test-vm-%d", r.Intn(10000))
+	for _, pvc := range pvcs {
+		vols = append(vols, vmopv1.VirtualMachineVolume{
+			Name: pvc.Name,
+			PersistentVolumeClaim: &vmopv1.PersistentVolumeClaimVolumeSource{
+				PersistentVolumeClaimVolumeSource: v1.PersistentVolumeClaimVolumeSource{ClaimName: pvc.Name},
+			},
+		})
+	}
+	labels := make(map[string]string)
+	labels["topology.kubernetes.io/zone"] = zone
```

Comment:
Please use tkgHATopologyKey instead of topology.kubernetes.io/zone

Reply:
tkgHATopologyKey                           = "topology.kubernetes.io"
can't use that